BDArmory/Damage/HitpointTracker.cs

```diff
@@ -881,6 +887,12 @@ public float CalculateTotalHitpoints()
                         }
                         if ((isProcPart || isProcWing) && BDArmorySettings.RUNWAY_PROJECT && BDArmorySettings.MAX_PWING_HP >= 100) hitpoints = Mathf.Clamp(hitpoints, 100, BDArmorySettings.MAX_PWING_HP);
 
+                        //if (hitpoints > 2000) //Hardcoded? slider setting? .cfg setting?
```

Comment:
I think this limit should be a slider setting and replace the `MAX_PWING_HP` clamp since there's not really any need for clamping if the HP is scaling logarithmically.